cms/tests/test_user_tasks.py

```diff
@@ -0,0 +1,95 @@
+"""
+Unit tests for integration of the django-user-tasks app and its REST API.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
Because I was using it for debugging, and it felt silly to keep taking it out and adding it back whenever I needed to track down a problem.  Also, the current draft of [OEP-7](https://github.com/edx/open-edx-proposals/pull/21/files) recommends having it in all files by default.